mlrun/config.py

```diff
@@ -796,7 +796,15 @@ def update(self, cfg, skip_errors=False):
         for key, value in cfg.items():
             if hasattr(self, key):
                 if isinstance(value, dict):
-                    getattr(self, key).update(value)
+                    try:
+                        getattr(self, key).update(value)
+                    except AttributeError as exc:
+                        config_value = getattr(self, key)
+                        if not isinstance(config_value, (dict, Config)):
```

Comment:
I think for completeness you need an else clause that raises the original exception